Makefile

```diff
@@ -0,0 +1,21 @@
+PREFIX=/usr/
+DESTDIR=/
+BINARY_NAME=apx
+
+all: build
+
+build:
+	go build -o ${BINARY_NAME}
```

Comment:
if you swap the below line, it'll be a static build and you can use standard Go or nix packaged Go:
`CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -tags netgo -ldflags '-w -extldflags "-static"'`

Reply:
Applied the change in https://github.com/Vanilla-OS/apx/pull/158/commits/bb3b7d573a60065595ba3bdc9e35388f0b2dda2e